docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version:  '3.7'
+
+services: 
```

Comment:
would be nice to get some docs for how to use this!

Reply:
created!